content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -88,6 +88,7 @@ const tracer = require('dd-trace').init({
   debug: true
```

Comment:
This should also be removed since this option no longer exists.

Reply:
There are still other occurrences of this in the file.

Reply:
Removed the other occurrences of the `debug:true `,